aten/src/ATen/FunctionalInverses.cpp

```diff
@@ -0,0 +1,191 @@
+
+#include <ATen/FunctionalInverses.h>
```

Comment:
Top of this file would be a good place to explain what a "view inverse" is

Reply:
In particular, there are some equational laws inverses should obey with their inputs, and this law is not exactly the same as that of a mathematical true inverse, so it's important to spell it out explicitly.